test/mini-testium-mocha.js

```diff
@@ -9,9 +9,15 @@ const createDriver = require('../');
 const browser = {};
 exports.browser = browser;
 
-browser.beforeHook = () => () =>
-  getTestium({ driver: createDriver }).then(testium => {
-    browser.__proto__ = testium.browser;
-  });
+browser.beforeHook = () => {
+  const currentTest = new Error().stack
```

Comment:
Should we just call this "testium-before-hook"? This seems rather fragile. Maybe if we use a function and `this` we can get this info from `mocha`'s API in a clean way?

Reply:
this is just the test faux-mocha thingy, but I shall poke for a better way